webapp/src/main/webapp/index.htm

```diff
@@ -38,7 +38,11 @@
         	<td width="100px"><div class="logo"></div></td>
         	<td>
             	<div class="bar-round">
-        			<div class="bar">OpenMRS Platform ${project.version}.${revisionNumber} Running!</div>
+        			<div class="bar">
+        				<h3>OpenMRS Platform ${project.version}.${revisionNumber} Running! </h3>
+        				<p>If you seeing this page, it means that the OpenMRS Platform is running successfully, but no user interface module is installed. To learn about the available user interfaces, see [<a href="https://wiki.openmrs.org/display/docs/Administering+Modules">https://wiki.openmrs.org/display/docs/Administering+Module/</a>]</p>
```

Comment:
@djazayeri I am aware that "Administering Modules" is the wrong link. I google for any openmrs wiki that was already built for that and the closest i could find was "Administering Modules". How ever i posted a comment on jira, saying that "Administering Module" link needs to be changed or the wiki modified. @djazayeri i would like to you give me all the information that you think should be on the wiki and also instructions on how to create the wiki. :-)